harnessed_jobs/linearity_sweep_raft_acq/v0/ccs_linearity_sweep_raft_acq.py

```diff
@@ -0,0 +1,100 @@
+"""
+Jython script to run output drain and reset drain voltage sweep acquisitions at TS8.
+"""
+
+from eo_acquisition import EOAcquisition, PhotodiodeReadout, AcqMetadata, logger
+import numpy as np
+import itertools
+
+class LinearitySweepAcquisition(EOAcquisition):
+    """
+    EOAcquisition subclass to sweep voltages and take flat image dataset.
+    """
+
+    def __init__(self, seqfile, acq_config_file, metadata, subsystems,
+                 ccd_names, logger=logger):
+
+        super(LinearitySweepAcquisition, self).__init__(seqfile, acq_config_file,
+                                                           "LINEARITYSWEEP", metadata,
+                                                           subsystems, ccd_names,
+                                                           logger=logger)
+
+    def run(self):
+        """
+        Take flat datasets at each voltage setting in accordance to the
+        configuration file.
+
+        Sample config:
+
+        linearitysweep 60 10 25.0 28.0 0.5 12.0 14.0 0.5 
+        """
+
+        openShutter = True
+        actuateXed = False
+        image_type = "RAMP"
+
+        seqno = 0
+        for tokens in self.instructions:
+            rowdelay = int(tokens[1])
+            nframes = int(tokens[2])
+            min_vod = float(tokens[3])
+            max_vod = float(tokens[4])
+            step_vod = float(tokens[5])
+            min_vrd = float(tokens[6])
+            max_vrd = float(tokens[7])
+            step_vrd = float(tokens[8])
+            test_type = "LINEARITYSWEEP"
```

Comment:
This is set in the constructor and does not need to be set here or below in `take_image`.